saleor/static/dashboard-next/products/components/ProductPrice/ProductPrice.tsx

```diff
@@ -0,0 +1,97 @@
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import { withStyles } from "@material-ui/core/styles";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableRow from "@material-ui/core/TableRow";
+import * as React from "react";
+
+import Money from "../../../components/Money";
+import PageHeader from "../../../components/PageHeader";
+import i18n from "../../../i18n";
+import Skeleton from "../../../components/Skeleton";
+
+interface MoneyType {
+  currency: string;
+  amount: number;
+}
+interface ProductPriceProps {
+  purchaseCost: {
+    start: MoneyType;
+    stop: MoneyType;
+  };
+  margin: {
+    start: number;
+    stop: number;
+  };
+}
+
+const decorate = withStyles(theme => ({
+  root: {
+    "& td, & th": {
+      paddingRight: theme.spacing.unit * 3
+    }
+  },
+  textRight: {
+    textAlign: "right" as "right"
+  }
+}));
+const ProductPrice = decorate<ProductPriceProps>(
+  ({ classes, margin, purchaseCost }) => (
+    <Card>
+      <PageHeader title={i18n.t("Pricing")} />
+      <Table className={classes.root}>
+        <TableBody>
+          <TableRow>
+            <TableCell>{i18n.t("Purchase cost")}</TableCell>
+            <TableCell className={classes.textRight}>
+              {purchaseCost ? (
+                purchaseCost.start.amount === purchaseCost.stop.amount ? (
+                  <>
+                    <Money
+                      amount={purchaseCost.start.amount}
+                      currency={purchaseCost.start.currency}
+                      typographyProps={{
+                        component: "span",
+                        style: { display: "inline" }
+                      }}
+                    />
+                    {" - "}
```

Comment:
We could have a dedicated price range widget.

Reply:
If I spot another place with ranged price, I'll do it.

Reply:
I'm saying this as I'd like to be able to change this depending on locale. For example typical English typography would have this be:

$ 22.00–25.00

rather than:

22.00 USD - 25.00 USD

Reply:
Okay, didn't know that it would matter. I'll write a ticket about it.